frontend/src/core/components/Dashboard/AppSidebar/AppSidebar.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from "react";
+import Sidebar from "../../Sidebar";
+import {Row} from "react-bootstrap";
+import MenuItem from "./MenuItem/MenuItem";
+import "./AppSidebar.scss";
+
+class AppSidebar extends Component {
+  render() { 
+    return <React.Fragment>
```

Comment:
Why **Fragment** here? this has more sense on **MenuItem** not here.

Here, we have the **Sidebar** element as root for this component.

Reply:
@justaname94 What about this one?

Reply:
I changed it on 08a7289